nni/experiment/config/utils/internal.py

```diff
@@ -15,7 +15,7 @@
     'fields', 'is_instance', 'validate_type', 'is_path_like',
     'guess_config_type', 'guess_list_config_type',
     'training_service_config_factory', 'load_training_service_config',
-    'get_ipv4_address'
+    'get_ipv4_address', 'init_experiment_config', 'get_experiment_class_using_config'
```

Comment:
Suggest to put these two functions below `training_service_config_factory` and `load_training_service_config`. They have somewhat similar purpose.

Reply:
And to make them aligned, `init_experiment_config` should be named `load_experiment_config`.

Reply:
updated